Maquetado/scss/details.scss

```
@@ -94,40 +112,77 @@
 }
 
 .book-title {
+  align-self: end;
+  grid-area: title;
   @include text-format(#{$big-font-size + 12}, $font-weight, #{$line-height-base + 18});
+  justify-self: center;
   margin-right: 5px;
 }
 
 .book-gender {
+  align-self: start;
+  grid-area: gender;
   @include text-format(#{$big-font-size + 4}, $font-weight, #{$line-height-base + 8});
+  justify-self: center;
 }
 
 .item-key {
-  @include text-format($big-font-size, $font-weight, $line-height-base);
+  @include text-format(#{$big-font-size + 4}, $font-weight, $line-height-base);
   margin-right: 5px;
 }
 
 .item-value {
-  font-size: $big-font-size;
+  font-size: #{$big-font-size + 4};
   line-height: #{$line-height-base + 4};
 }
 
 .list {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-gap: 50px;
+  grid-template: repeat(2, 275px) / repeat(4,165px);
+  grid-template-areas: 'a''b''c''d''e''f''g''h';
```

The structure must be 4 rows, so replace this by: 
```suggestion
  grid-template-areas: 'a b c d' 'e f g h';
```